pull_request_template.md

```diff
@@ -1,20 +1,51 @@
 
-- [ ] Addition
-- [ ] Removal
-- [ ] Edit
-- [ ] Other
+- [ ] **Adding** something new to awesome-ipfs
+- [ ] **Editing** something already listed on awesome-ipfs
+- [ ] **Removing** something from awesome-ipfs
+- [ ] **Something else** *(if so, please explain in the "Additional details" section below)*
 
 <!-- If your change is not listed above, please remove the checklist bellow. -->
 
-### Checklist
+## Pre-submit checklist
+**Please confirm ALL of the following** before submitting your PR.
 
+- [ ] This PR includes only one addition, removal, or edit.
 - [ ] I edited the `/data` directory instead of the [README.md](https://github.com/ipfs/awesome-ipfs/blob/master/README.md).
-- [ ] This PR includes only one addition/removal/edit.
 - [ ] I ran the `make build` command following my edits to the `/data` directory.
-- [ ] I reviewed the [content policy](https://github.com/ipfs/awesome-ipfs/blob/master/POLICY.md) to ensure my submission meets the requirements.
+- [ ] I reviewed the [content policy](https://github.com/ipfs/awesome-ipfs/blob/master/POLICY.md) and the and the [IPFS Community Code of Conduct](https://github.com/ipfs/community/blob/master/code-of-conduct.md) to ensure my submission meets the requirements.
 - [ ] I have followed the [CONTRIBUTING.md guidelines](https://github.com/ipfs/awesome-ipfs/blob/master/CONTRIBUTING.md).
 
-## Details
+## Additional details
 
-Provide details of your changes here.
+### Is there anything else we should know about this PR?
+*If you've checked "Something else" above, or just want to provide additional info or clarification, please do so here.*
+
+### For new additions: What is your project's *current* size/reach?
+*This could be number of users, number of integrations, frequency of use, or any other key metrics. If your PR isn't for a new addition to awesome-ipfs, or you don't know these numbers, you can leave this out.*
+
+### For new additions: What is your project's *potential* size/reach?
+*This could be number of users, number of integrations, frequency of use, or any other key metrics. If your PR isn't for a new addition to awesome-ipfs, or you don't know these numbers, you can leave this out.*
+
+### For new additions: How critical is IPFS to your project?
+*Choose one. If your PR isn't for a new addition to awesome-ipfs, you can leave this out.*
+- [ ] Essential
+- [ ] Somewhat critical
+- [ ] Useful, but not critical
+
+### For new additions: What core goal(s) does your project address?
+*Choose as many as apply. If your PR isn't for a new addition to awesome-ipfs, you can leave this out.*
+- [ ] Big-data solutions
+- [ ] Freedom from corporate/government interference
+- [ ] Data integrity
+- [ ] Dev tools or other ways to enable developers
+- [ ] Disaster resilience/recovery
+- [ ] File storage/retrieval
+- [ ] File streaming
+- [ ] Permanent archiving
+- [ ] Self-sovereign idenity
+- [ ] Platforms that utilize consensus, reputation, or incentivization
+
```

Comment:
Might also want an "Other" option on the end here?

Reply:
Good call.